pynpoint/util/tests.py

```diff
@@ -299,6 +299,58 @@ def create_star_data(path: str,
         hdu.writeto(os.path.join(path, f'images_{j}.fits'))
 
 
+@typechecked
```

Comment:
Didn't we already add a function for test IFS data in the previous PR?

Reply:
Yes we added a IFS data function for the PSFsubtractionModule unit tests. But since non-IFS test data was also generated with two functions, I did it accordingly.

Reply:
Let's try to use the function that was already there.

Reply:
I'll removed `create_star_data_ifs` and used `create_ifs_data` instead.